Adieu razzle (#87)
deity-io/falcon: packages/falcon-client/README.md

The build is minified and the filenames include the hashes. Your app is ready to be deployed!

### `falcon-client size`

Runs an interactive zoomable treemap of output files to visualize their size. Report will be automatically open in default browser at `http://localhost:8888`.

## API contract

> Application needs to have `index.js`, and following optional configuration files `bootstrap.js` and `falcon-client.build.config.js`. Each of them should be placed in application root directory.

Reviewer: ```suggestion
Application needs to have `index.js` file, and the following optional configuration `bootstrap.js` and `falcon-client.build.config.js`  files. Each of them should be placed into the application root directory.
```

- `process.env.PORT`- (default: `3000`), builded in only when `development`
- `process.env.HOST`- default is `0.0.0.0`
- `process.env.BUILD_TARGET` - `client` or `server`
- `process.env.ASSETS_MANIFEST` - path to webpack assets manifest,
- `process.env.PUBLIC_DIR`: directory with your static assets

You can create your own custom build-time environment variables. They must be listed on `envToBuildIn` in `falcon-client.build.config.js` file. Any other variables except the ones listed above will be ignored to avoid accidentally exposing a private key on the machine that could have the same name. Changing any environment variables will require you to restart the development server if it is running.

For example, bellow configuration expose environment variable named `SECRET_CODE` as `process.env.SECRET_CODE`:

```js
module.exports = {
  envToBuildIn: ['SECRET_CODE']
};
```

### Webpack

Falcon Client gives you the possibility to extend the underlying webpack configuration. You can do it via exposed plugins API (not webpack plugins). It is worth to mention that Falcon Client plugins API is [razzle](https://github.com/jaredpalmer/razzle#plugins) compatible.

> To use Falcon Client (or Razzle) plugin, you need to instal it in your project, and add it to `plugins` array in `falcon-client.build.config.js` file:

Reviewer 2: ```suggestion
To use Falcon Client (or Razzle) plugin, you need to install it in your project, and add it to `plugins` array in `falcon-client.build.config.js` file:
```

- `onServerInitialized(server: Koa)` - handler invoked immediately after koa server setup (when middlewares like handling errors, serving static files and routes were set up)
- `onServerStarted(server: Koa)` - handler invoked when koa server started with no errors

### `falcon-client.build.config.js`

This is an optional build time configuration file which is used to setting up entire build process.

> By default support for following is turned on:

Reviewer: Not sure if I get this sentence right.

```suggestion
By default it supports the following features:
```
